README.md

```diff
@@ -10,3 +10,10 @@ ami JWT er ekta full flow implement korar cheshta korsi. ei koyta jinish eikhane
     - refresh-token diye new JWT anbo, and `RS` & `LS` a rakhbo
     - j koyta dead-call hobe, oi call gula abar korte hobe.
     - suppose 4 ta dead call holo, 4 baar refresh-token er api call hobe na (async-mutex)
+
+
+
+
+## Run Local Json Server
+
+npx json-server --watch db.json --port 5000
```

Comment:
pls add this into [package.json ](https://github.com/saiffardin/rtk-query-jwt/blob/4c06551e0753178bfe17a7bf6382e56a1a5c6c7c/package.json) file

```
"scripts": {
   // previous scriptsh
    "server": "npx json-server --watch db.json --port 5000"
  },
```